app/controllers/movies_controller.rb

```diff
@@ -0,0 +1,42 @@
+# frozen_string_literal: true
+
+class MoviesController < ApplicationController
+  def index
+    @movies = Movie.all.map do |movie|
+      movie_params(movie)
+    end
+    render json: @movies
+  end
+
+  def show
+    movie = Movie.find(params[:id])
+    @transformed_movie = movie_params(movie)
+    render json: @transformed_movie
+  end
+
+  def create
+    Movie.create(movie_params)
+  end
+
+  def update
```

Comment:
You can respond with a proper response code after a successful action. Same for the `delete` action.